pkg/syncer/fullsync.go

```diff
@@ -82,6 +74,17 @@ func CsiFullSync(ctx context.Context, metadataSyncer *metadataSyncInformer) erro
 			k8sPVMap[pv.Spec.CSI.VolumeHandle] = ""
 		} else if migrationFeatureStateForFullSync && pv.Spec.VsphereVolume != nil {
 			// For vSphere volumes, migration service will register volumes in CNS.
+
+			// Note that we can never reach here in case of a multi VC setup
```

Comment:
Do you mean k8sPVs will not have in-tree vSphere volumes when `len(metadataSyncer.configInfo.Cfg.VirtualCenter) > 2` ?

Can you point me to where that code is?

Reply:
See this line where I am taking care of in-tree volumes: https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/2087/files#diff-bf0a9ad2d6dc8555d4661f433fe187035ab27a29ea3bf395e23713f69a0bed23R564